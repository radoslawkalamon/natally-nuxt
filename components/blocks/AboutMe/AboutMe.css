.about-me {
  display: grid;
  gap: var(--gap);
  grid-template-areas:
    "image"
    "text";
  justify-items: center;
}

@media screen and (min-width: 480px) {
  .about-me {
    align-items: center;
    grid-template-areas: "image text";
    justify-items: stretch;
  }
}

.about-me__image {
  border-radius: var(--border-radius);
  grid-area: image;
  height: 120px;
  width: 120px;
}

.about-me__text {
  grid-area: text;
}
