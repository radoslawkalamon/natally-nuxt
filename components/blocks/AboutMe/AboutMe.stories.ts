import type { Meta, Story } from '@storybook/vue'
import BlocksAboutMe from '@/components/blocks/AboutMe/AboutMe.vue'

const meta: Meta = {
  title: 'Blocks / About Me',
  component: BlocksAboutMe
}
export default meta

export const Default: Story = () => ({
  components: {
    BlocksAboutMe
  },
  template: '<div class="__storybook-section__"><BlocksAboutMe /></div>'
})
