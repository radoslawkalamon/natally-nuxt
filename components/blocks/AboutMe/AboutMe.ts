import Vue from 'vue'

export default Vue.extend({
  name: 'BlocksAboutMe',
  components: {
    WrappersText: () => import(/* webpackChunkName: "wrappers" */'@/components/wrappers/Text/Text.vue')
  }
})
