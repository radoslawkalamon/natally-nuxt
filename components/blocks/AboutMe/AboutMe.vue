<template>
  <div
    v-once
    class="about-me"
    data-test="blocks-about-me"
  >
    <img
      alt="Zdjęcie przedstawia Radosława Kalamona"
      class="about-me__image"
      height="120"
      src="./images/radoslaw-kalamon.1x.webp"
      srcset="./images/radoslaw-kalamon.1x.webp 1x, ./images/radoslaw-kalamon.2x.webp 2x"
      width="120"
    >
    <WrappersText class="about-me__text">
      <p>
        Cześć! Jesteś na blogu 169cm.pl. Ja jestem Radosław Kalamon. Chodź ze mną. Usiądźmy przy ognisku i spójrzmy do góry w czarne niebo upstrzone tysiącem białych punktów. Ty zacznij wypatrywać spadających gwiazd, a ja opowiem Ci historię o jednej z nich.
      </p>
    </WrappersText>
  </div>
</template>

<script src="./AboutMe.ts" lang="ts"></script>
<style src="./AboutMe.css" scoped></style>
