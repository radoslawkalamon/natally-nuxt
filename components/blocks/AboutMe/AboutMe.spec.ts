import AboutMe from '@/components/blocks/AboutMe/AboutMe.vue'
import { shallPassIntegrationSanityTest } from '@/devtools/jest.common.spec'

describe('Blocks / About Me', () => {
  describe('Integration', () => {
    shallPassIntegrationSanityTest({ component: AboutMe })
  })
})
