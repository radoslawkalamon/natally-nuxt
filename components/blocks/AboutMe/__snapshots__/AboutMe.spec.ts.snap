// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blocks / About Me Integration shall pass integration sanity test 1`] = `
"<div class=\\"about-me\\"><img alt=\\"Zdjęcie przedstawia Radosława Kalamona\\" height=\\"120\\" src=\\"./images/radoslaw-kalamon.1x.webp\\" srcset=\\"./images/radoslaw-kalamon.1x.webp 1x, ./images/radoslaw-kalamon.2x.webp 2x\\" width=\\"120\\" class=\\"about-me__image\\">
  <div class=\\"about-me__text text\\">
    <p>Cześć! Jesteś na blogu 169cm.pl. Ja jestem Radosław Kalamon. Chodź ze mną. Usiądźmy przy ognisku i spójrzmy do góry w czarne niebo upstrzone tysiącem białych punktów. Ty zacznij wypatrywać spadających gwiazd, a ja opowiem Ci historię o jednej z nich.</p>
  </div>
</div>"
`;
